src/pivotal-ui/components/pui-variables.scss

```diff
@@ -321,6 +321,7 @@ $font-family-sans-serif:  'Open Sans', 'OpenSans', sans-serif !default;
 $font-family-serif:       Georgia, "Times New Roman", Times, serif !default;
 $font-family-monospace:   Menlo, Consolas, "Courier New", monospace !default;
 $font-family-base:        $font-family-sans-serif !default;
+$font-cap-height-ratio:   0.73;
```

Comment:
Me visually measuring the type of OpenSans. Though now looking at it again, we're then defining other fonts right there and it's all stupidly fraught.